modules/govuk/manifests/apps/efg_training.pp

```diff
@@ -28,7 +28,7 @@
 ) {
   validate_string($vhost_name)
 
-  $app_name = 'efg_training'
+  $app_name = 'efg-training'
```

Comment:
Your commit was correct compared to the previous behaviour! It made the `$app` parameter of the `govuk::app::envvar` resource be set to the title of a `govuk::app` resource which existed, rather than one that didn't. But I think both of those should be a hyphen rather than an underscore.